### PR TITLE
tls partial chain

### PR DESCRIPTION
**Pull request by qzhuyan**

### CODE REVIEW

File: en_US/configuration/configuration.md [hunk 1]
```diff
@@ -2947,6 +2947,24 @@ Specifies whether to verify the client during the handshake.
...
+
+| Type | Optional Value                                                           | Default |
+|------|--------------------------------------------------------------------------|---------|
+| enum | `false`, `true`, `cacert_from_cacertfile`, `two_cacerts_from_cacertfile` | `false` |
+
+#### Description
+
+When EMQX verifies a client certificate during the x509 path validation process, it constructs a certificate chain that starts with the client certificate and ends with a trust anchor.
+
+By default, if the setting is set to `false`, the trust anchor is the rootCA, and the certificate chain must be complete.
```
Comment (kjellwinblad) [inline en_US/configuration/configuration.md:2960]: Just out of curiosity. Is rootCA provided by the operating system or where does it come from?

**Review by kjellwinblad — APPROVED** [review]

File: en_US/configuration/configuration.md (hunk 1 above)
Comment (qzhuyan) [reply en_US/configuration/configuration.md:2960]: from `cacertfile`

**Comment by hjianbo** [comment]
nit:  Do we need to update the Chinese documentation at the same time?

**Review by thalesmg — APPROVED** [review]

**Comment by qzhuyan** [comment]
> nit: Do we need to update the Chinese documentation at the same time?

We now have technical writer 😸

**Pull request merged**